cloudformation/PolicyTemplates/03-cross-account-policy.yml

```diff
@@ -0,0 +1,24 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Allows users in parsys or infosec prod to call assumeRole for CISDeveloper in infosec-dev."
```

Comment:
There is one permission that you have to place on Parsys accounts if non admin that allows them to assume the role.  Everything else is already setup with trust to PartInfra.